posthog/api/person.py

```diff
@@ -42,7 +42,11 @@ class CursorPagination(BaseCursorPagination):
 
 
 class PersonViewSet(viewsets.ModelViewSet):
-    renderer_classes = tuple(api_settings.DEFAULT_RENDERER_CLASSES) + (csvrenderers.PaginatedCSVRenderer,)
+    _PROD_DISTINCT_ID_REGEX_PATTERN = r"[A-Za-z0-9]{14}-[A-Za-z0-9]{14}-[A-Za-z0-9]{8}"
```

Comment:
Yeah the reason I had a more "liberal" regex pattern was that on all my "anonymous" users, the hyphen after 14 alphanumeric characters was consistent, but the rest of the ID had hyphens at varying positions. Just tested this and this regex misidentifies some people for me. 

Will become irrelevant anyway though, given Tim's new suggested solution.